portfolio.html

```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder5">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder6">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder7">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+        </div>
+    </section>
+
+    <section class="about-me">
+        <div class="text-about-me">
+            <h1 class="about">About me</h1>
+            <img src="./images/mobile_version/Card/illustration-1-about-me-mobile.png" class="about-me-mobile" alt="illustration icon">
+            <p class="about-me-text"> I can help you build a product , <br> feature or website Look through <br> some of my work and experience! If <br> you like what you see and have a <br> project you need coded, <br>
+                don’t hestiate to contact me.</p>
+            <button type="button" class="get-my-resume">Get My Resume</button>
```

Comment:
- [ ] Kindly add the appropriate button  styling for  all buttons in this section, please see the design, and follow the button styling for Active, Hover, and Pressed
- [ ] Please remove the border outline of all buttons, this will make your buttons look exactly like the Figma design 👍🏿 
![Screenshot_395](https://user-images.githubusercontent.com/69638013/185760017-d8da5417-e31e-4b7d-bb3d-f5bfcf36e283.png)